module-2/lab-subsetting-and-descriptive-stats/your-code/main.ipynb

```diff
@@ -21,7 +21,7 @@
   },
```

Comment:
Great that you have used data manipulation functions **"Find the minimum, mean and the maximum of all numeric columns for each Department.Hint: Use functions from Data Manipulation lesson"** instead of describe function. 
You can skip the first column of orders csv (the one pandas read_csv calls unnamed) in order to avoid further misunderstanding. 
It's a good practice when importing data sets to check data types in case you don't specify that types in read_csv function. It could be a good idea to convert the InvoiceDate variable into a datetime type and also to split date and time in 2 columns. 
Take a look on this question "Select orders from Spain that are above 50 items". I think you are missing the conditional. 
Take a look on this question Select all free orders. I think you are missing the conditional for getting only free orders. 
When creating bins in an histogram it's a good practice not to include a high number of bins. It's more difficult to read the plot and to interpret it.